client/lib/faux-plugins.js

```diff
@@ -224,7 +238,7 @@ module.exports = function(roomName) {
         })
         .filter(Boolean)
 
-      var playRe = /!play [^?]*\?v=([-\w]+)/
+      var playRe = /!play [^?]*\?v=([-\w]+)(&t=([0-9hms]+))?/
```

Comment:
Small suggestion: make the surrounding group non-capturing:

    var playRe = /!play [^?]*\?v=([-\w]+)(&t=(?:[0-9hms]+))?/